### PR TITLE
Updated slingshot critical telemetry message ID name

### PR DESCRIPTION
**Pull request by danielvaz46**
### Summary and Scope

The message id of the slingshot fabric critical telemetry data has been updated to CrayFabricCriticalTelemetry from CrayFabricCritTelemetry.

### CODE REVIEW

**Comment by rsjostrand-hpe** [comment]
Also FYI @mjendrysik-hpe @nieuwsma this is a PR to merge changes into the [integration/ss_health_topics](https://github.com/Cray-HPE/hms-hmcollector/tree/integration/ss_health_topics) branch.

File: .version [hunk 1]
```diff
@@ -1 +1 @@
-2.16.0
+2.16.1
```
[the remark this answers is not in the document]
Comment (nieuwsma) [reply .version:1]: yeah; but WE (HMS) can reset that version number when we go from integration to master...

**Comment by nieuwsma** [comment]
@danielvaz46 when did Slingshot roll this change out? what Shasta release?   Is the currently broken in LIME or STRAWBERRY?   We need to make sure we braking a production version.

**Comment by danielvaz46** [comment]
> @danielvaz46 when did Slingshot roll this change out? what Shasta release? Is the currently broken in LIME or STRAWBERRY? We need to make sure we braking a production version.

Hi @nieuwsma, 
Sorry for the delayed response.
Slingshot rolled out this change in Slingshot 1.5 released in October.
It is currently broken in LIME

**Comment by rfrost-hpe** [comment]
@danielvaz46  is there a JIRA for this change?

**Comment by danielvaz46** [comment]
> @danielvaz46 is there a JIRA for this change?

Yes. https://jira-pro.its.hpecorp.net:8443/browse/CASMSMF-6765

**Comment by danielvaz46** [comment]
@nieuwsma I have tested out the change on frigg. Below is a screenshot of the data now flowing into kafka.
image
Do let me know any additional details are required. Otherwise, can we proceed with the merging of this PR.
Thanks

**Comment by alvarez3-hpe** [comment]
This change has been merged with:
https://github.com/Cray-HPE/hms-hmcollector/pull/36

**Pull request closed without merging**